src/util/system.cpp

```diff
@@ -1126,6 +1127,15 @@ fs::path GetSpecialFolderPath(int nFolder, bool fCreate)
 }
 #endif
 
+#ifndef WIN32
```

Comment:
Maybe leaving the UNIX quoting in place is better than nothing at all?

Reply:
> Maybe leaving the UNIX quoting in place is better than nothing at all?

I think there was already agreement to not support `%w` on windows, and leave windows escaping for a future PR: https://github.com/bitcoin/bitcoin/pull/13339#issuecomment-537384875, instead of implementing something broken. This is the simplest and least dangerous approach.

Reply:
Right. Let's please not repeat this argument. If we do, this will never make any progress.